src/main/java/cloud/fogbow/common/constants/CloudStackConstants.java

```diff
@@ -136,9 +135,13 @@
     }
     
     public static class SecurityGroupPlugin {
+      public static final String CREATE_FIREWALL_RULE_RESPONSE = "createfirewallruleresponse";
+      public static final String DELETE_FIREWALL_RULE_RESPONSE = "deletefirewallruleresponse";
     	public static final String LIST_FIREWALL_RULES_COMMAND = "listFirewallRules";
-    	public static final String IP_ADDRESS_ID_KEY_JSON = "ipaddressid";
+      public static final String CREATE_FIREWALL_RULE_COMMAND = "createFirewallRule";
+      public static final String DELETE_FIREWALL_RULE_COMMAND = "deleteFirewallRule";
     	public static final String LIST_FIREWALL_RULES_KEY_JSON = "listfirewallrulesresponse";
+    	public static final String IP_ADDRESS_ID_KEY_JSON = "ipaddressid";
```

Comment:
Fix the indentation of these constants in this inner class.

Reply:
Done